### ISSUE TITLE
Валидация у Drag'n'DropField

### ISSUE THREAD

**Issue by hidmitry** [consta-design-system/uikit#1833]
В библиотеке валидация выглядит так:

<img width="398" alt="Screenshot 2021-11-29 at 20 00 10" src="https://user-images.githubusercontent.com/95356759/145054320-734a4f8e-03d0-45c4-b634-1f3e828e574e.png">

У нас в старых макетах был такой вариант:

<img width="512" alt="Screenshot 2021-11-29 at 20 03 45" src="https://user-images.githubusercontent.com/95356759/145054388-102426d7-d300-445d-8c83-7e01d7a61f40.png">

Я собрал компромиссный вариант:

<img width="449" alt="Screenshot 2021-11-29 at 20 00 17" src="https://user-images.githubusercontent.com/95356759/145054452-b121c656-69b9-437e-b5cb-06e95f55ad92.png">

На мой взгляд, самый удачный вариант был у нас в старых макетах, потому что зона для загрузки файла визуально все так же доступна для загрузки, с каким именно файлом возникла проблема тоже видно. А вот библиотечный вариант выглядит так, как будто сама форма сломалась и какой файл пытались загрузить непонятно. 

В моем варианте первый пункт я снял, но непонятно с каким именно файлом возникла проблема. Как вариант название файла можно отображать в тултипе.

**Comment by Ilyashis** [comment]
Компонент использован неправильно.
Загружаемые файлы отображаются компонентом Attachment под формой Drag'n'DropField
Ошибки при загрузке отображаются в Attachment

**Comment by Ilyashis** [comment]
Пример показал - https://www.figma.com/file/v9Jkm2GrymD277dIGpRBSH/Consta-UI-Kit?node-id=32774%3A0

**Comment by hidmitry** [comment]
@Ilyashis, спасибо. 
Хотел бы уточнить, в каких случаях используется поле с ошибкой, которую я принял за ошибку валидации? Может, отобразить этот сценарий в тексте ошибки, чтобы это было более очевидно?

**Comment by Ilyashis** [comment]
Согласен, в случаи множественной загрузки очевиднее было бы показать ошибку о формате в списке файлов.

**Comment by linturaven** [comment]
https://github.com/consta-design-system/uikit/issues/3039 не забыть про это issue

**Comment by hell0anna** [comment]
жду ревью ишью https://github.com/consta-design-system/uikit/issues/3039

**Comment by linturaven** [comment]
Обсудили 21 фев 2024, что надо сделать:

**Дизайн**
- [ ] проработать состояние ошибки в дизайне (подумать над статусностью и функциональностью полосы ошибки)
- [ ] подумать над отображением прогресса загрузки
- [ ] собрать компоненты в фигме
- [ ] завести issue по отображению файлов в dnd

**Дока**
- [ ] описать рекомендации по бизнес-логике, в первую очередь, по пакетной загрузке файлов

**Разработка**
- [ ] учесть, что API компонента должен позволить пользователю выводить ошибку в нужном ему формате


На выходе получим dragndrop с статусным caption, который можно включать и выключать и прокидывать ошибки в своём виде, в зависимости от бизнес-логики.

**Comment by hell0anna** [comment]
[готово](https://www.figma.com/file/v9Jkm2GrymD277dIGpRBSH/Consta-UI-Kit?type=design&node-id=34445%3A0&mode=design&t=Hsfqz0sJLMHNT1Ch-1)
